sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -806,6 +808,7 @@ This project has adopted the [Microsoft Open Source Code of Conduct][code_of_con
 [labeling_tool]: https://aka.ms/azsdk/formrecognizer/labelingtool
 [dotnet_lro_guidelines]: https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-longrunning
 
+[troubleshooting]: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/formrecognizer/Azure.AI.FormRecognizer/README.md
```

Comment:
I need to merge this PR before adding this link (will replace README.md -> TROUBLESHOOTING.md later).